### PR TITLE
Update deployment files

### LINKED ISSUE #158: Deployment of UI along with backend(controller)
**Is this a BUG REPORT or FEATURE REQUEST?**:
/kind feature

**What happened and What you expected to happen**:
- Avoid `kubectl proxy` for integrating UI and controller
- Expose ports externally only if necessary Ex: `dgraph` ports are not needed to be exposed externally
- Remove assumption of purser getting deployed in `purser` namespace
- Simplify deployment files

### PR DESCRIPTION
**Pull request by kreddyj**
**What this PR does / why we need it**:
Previously integrating UI with controller requires `kubectl proxy` and hence was not a smooth process.

With this PR we use NodePorts for purser controller and ui which allows us to access them using `externalIP:PortNumber`. It also removes unnecessary port exposure of dgraph ports.

This updated deployment file doesn't assume any particular namespace while the previous method assumed purser is going to be deployed in `default` namespace.

Now just one line change will allow user to run UI in development mode.

**Which issue(s) this PR fixes**:
Fixes #158

### CODE REVIEW

File: docs/sourcecode-installation.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ make push
 
 ### Install Purser Plugin
 
-- Update the image name in [`purser-controller-setup.yaml`](../cluster/purser-controller-setup.yaml) to the docker image name that you pushed.
+- Update the image name in [`purser-controller-setup.yaml`](../cluster/purser-database-setup.yaml) to the docker image name that you pushed.
```
Comment (hemani19) [inline docs/sourcecode-installation.md]: Update file name - `purser-controller-setup.yaml`

File: README.md [hunk 2]
```diff
@@ -64,31 +64,29 @@ Purser has three components to install.
...
+and `kubectl create -f purser-setup.yaml` will setup purser in your cluster.
 
 ##### Change Settings and Enable/Disable Purser Features
 
 The following settings can be customized before Controller installation:
 
-- Change the default **log level**, **dgraph url** and **dgraph port** by editing `args` field in the [purser-controller-setup.yaml](./cluster/purser-controller-setup.yaml). (Default: `--log=info`, `--dgraphURL=purser-db`, `--dgraphPort=9080`)
-- Enable/Disable **resource interactions** capability by editing `args` field in the [purser-controller-setup.yaml](./cluster/purser-controller-setup.yaml) and uncommenting `pods/exec` rule from purser-permissions. (Default: `disabled`)
+- Change the default **log level**, **dgraph url** and **dgraph port** by editing `args` field in the [purser-controller-setup.yaml](cluster/purser-database-setup.yaml). (Default: `--log=info`, `--dgraphURL=purser-db`, `--dgraphPort=9080`)
+- Enable/Disable **resource interactions** capability by editing `args` field in the [purser-controller-setup.yaml](cluster/purser-database-setup.yaml) and uncommenting `pods/exec` rule from purser-permissions. (Default: `disabled`)
```
Comment (hemani19) [inline README.md]: Update filenames

File: docs/sourcecode-installation.md (hunk 1 above)
Comment (kreddyj) [reply docs/sourcecode-installation.md]: Done

File: README.md (hunk 2 above)
Comment (kreddyj) [reply README.md]: Done

**Pull request merged**